app/views/practices/form/about.html.erb

```diff
@@ -13,6 +16,9 @@
       <%= render partial: "shared/messages", locals: {small_text: false} %>
       <section class="usa-section padding-top-0 p0-bottom about">
         <h1 class="font-sans-2xl line-height-46px margin-bottom-3">About</h1>
+        <div class="margin-top-2 margin-bottom-2">
+          <%= render partial: "practices/form/practice_last_updated", locals: {small_text: true} %>
+        </div>
```

Comment:
We can use the class margin-y-2 here, since both the top and bottom margin sizes are identical.